docs/tutorials/introduction-to-jax.ipynb

```diff
@@ -0,0 +1,646 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# A brief introduction to JAX\n",
+    "\n",
+    "Google JAX is a high-performance, open-source numerical computing library with a NumPy-like API that is notable for the following features.\n",
+    "- **XLA compilation:** JAX allows certain Python functions to be compiled more efficiently using XLA.\n",
+    "- **Hardware acceleration:** In addition to the CPU, JAX seamlessly integrates with GPUs and TPUs.\n",
+    "- **Differentiable Programming:** JAX comes with high-performance automatic differentiation.\n",
+    "\n",
+    "This tutorial is meant to give you a brief introduction to JAX and is structured as follows.\n",
+    "\n",
+    "- [An accelerated NumPy](#an-accelerated-numpy)\n",
+    "- [Useful higher-order functions](#useful-higher-order-functions)\n",
+    "- [PyTrees](#pytrees)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "(an-accelerated-numpy)=\n",
+    "## An accelerated NumPy\n",
+    "\n",
+    "A major appeal of JAX is that its API is largely based on the highly prevalent scientific computing library NumPy.\n",
+    "As a consequence, many programs would run just as well in JAX if we replaced `np` with `jnp`.\n",
+    "Let's look at a few lines of code written in JAX, to illustrate this:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import jax\n",
+    "import jax.numpy as jnp\n",
+    "import numpy as np\n",
+    "\n",
+    "x = jnp.linspace(0, 5, 11)\n",
+    "x"
```

Comment:
`print(x)`

Reply:
The reason why I preferred to display `x` instead of printing it is that displaying also shows the type `float32`, which `print(x)` would not. Why would you favour `print(x)`?